turbo.json

```diff
@@ -7,6 +7,10 @@
         "^build"
       ]
     },
+    "watch": {
+      "cache": true,
```

Comment:
I'm not entirely sure if in this context wouldn't be better to set `cache: false`. I tested both options in my environment and didn't saw a big change in behavior.